modern/src/App.js

```diff
@@ -273,6 +284,40 @@ function Button({
   );
 }
 
+function Popupmenu({ id, node }) {
+  const children = node.commands.map(item => {
+    if (item.id === "seperator") {
+      return <li />;
+    }
+    return (
+      <li
+        key={item.id}
+        onClick={() => {
+          node.resolveCmdSelection(item.id);
+        }}
+      >
+        {item.name}
+      </li>
+    );
+  });
+  const { x, y } = mouseposition;
```

Comment:
I would do this with a `useEffect` that listens for the document mousemove event and then calls a local setState. If it happens too frequently, we can throttle it.